modules/terrain/src/lib/parse-terrain.js

```diff
@@ -89,15 +116,49 @@ function getMartiniTileMesh(terrainImage, terrainOptions) {
       boundingBox: getMeshBoundingBox(attributes)
     },
     mode: 4, // TRIANGLES
-    indices: {value: triangles, size: 1},
+    indices: {value: Uint32Array.from(triangles), size: 1},
     attributes
   };
 }
 
+/**
+ * Get Martini generated vertices and triangles
+ *
+ * @param {number} meshMaxError threshold for simplifying mesh
+ * @param {number} width width of the input data
+ * @param {number[] | Float32Array} terrain elevation data
+ * @returns {object} vertices and triangles data
```

Comment:
From [the documentation](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns) looks like you should be able to do

```
 * @returns {{ vertices: Uint16Array, triangles: Uint32Array }}
```

which gives much better type information than just `object`